### ISSUE TITLE
There is no icon or name for Debugbot when you engage him on mobile

### ISSUE THREAD

**Issue by jessepickard** [debugpolitics/debugbot#2]
![img_0520 png](https://cloud.githubusercontent.com/assets/304836/21408330/6525dcbe-c789-11e6-95d9-bfcbd042b5aa.png)

**Comment by avand** [comment]
Have you tried quitting Slack and opening it again? I think that's a Slack bug. I've seen the before.

**Comment by jessepickard** [comment]
Yeah, I have. Also, sometimes when I type /do it returns "/do is not a valid command. In Slack..." See pic -->
![img_0521 png](https://cloud.githubusercontent.com/assets/304836/21411481/a9404fce-c79d-11e6-8d17-f3c7a92c077b.png)
